api/bertyverifiablecreds.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+
+package berty.account.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "berty.tech/berty/go/pkg/verifiablecredstypes";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+
+service VerificationCodeSender {
+  rpc SendVerificationCode (SendVerificationCode.Request) returns (SendVerificationCode.Reply);
+  rpc ValidateIdentifier (ValidateIdentifier.Request) returns (ValidateIdentifier.Reply);
+}
+
+message VerifiableCredsNonce {
+  bytes timestamp = 1;
+  bytes nonce = 2;
+  string berty_link = 3;
+  string redirect_uri = 4 [(gogoproto.customname) = "RedirectURI"];
+  string state = 5;
+}
+
+message VerifiableCredsContext {
+  bytes timestamp = 1;
+  string berty_link = 2;
+  CodeStrategy code_strategy = 3;
+  string identifier = 4;
+  string code = 5;
+  string redirect_uri = 6 [(gogoproto.customname) = "RedirectURI"];
+  string state = 7;
+}
+
+message AccountCryptoChallenge {
+  string challenge = 1;
+}
+
+message SendVerificationCode {
+  message Request {
+    string recipient = 1;
+    string code = 2;
+    string network_context = 3;
+    string context = 4;
+  }
+  message Reply {}
+}
+
+message ValidateIdentifier {
+  message Request {
+    string identifier = 1;
+  }
+  message Reply {
+    string sanitized = 1;
+  }
+}
+
+enum FlowType {
+  FlowTypeUndefined = 0;
+  FlowTypeCode = 1;
+  FlowTypeAuth = 2;
+  FlowTypeProof = 3;
```

Comment:
Can you specify that this mode are not implemented yet and declared only to be future proof?